ww3tools/ProcSat_Altimeter.py

```diff
@@ -347,13 +286,13 @@ def savesat(AODN,wconfig,altsel):
         smethod="AlongTrack"
         hmsg="AODN Altimeter data processed along track."
 
-    datein = datetime.utcfromtimestamp(AODN['TIME'].iloc[0]).strftime('%Y%m%d%H')
-    datefin = datetime.utcfromtimestamp(AODN['TIME'].iloc[-1]).strftime('%Y%m%d%H')
+    datein = datetime.utcfromtimestamp(AODN_ALONGTRACK['TIME'][0]).strftime('%Y%m%d%H')
```

Comment:
I don't see anything about this update in the text of the PR, can you tell me a little about why we changed from AODN to AODN_ALONGTRACK?